awesome_owl/static/src/counter/counter.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import { Component, useState } from "@odoo/owl";
+
+export class Counter extends Component {
+    static template = "awesome_owl.Counter";
+    static props = {
+        onChange: { type: Function, optional: true }
+    };
+
+    setup() {
+        this.state = useState({ value: 1 });
+    }
+
+    increment() {
+        this.state.value++;
+        if (this.props.onChange) {
+            this.props.onChange();
+        }
```

Comment:
```suggestion
            this.props.onChange?.();
```
Use optional chaining for compact code